src/Perpetuum/Zones/Terrains/TerrainUpdateNotifier.cs

```diff
@@ -166,7 +166,10 @@ private void DequeueNewUpdates()
                     }
 
                     var cell = _grid.GetCell(info.Position);
-                    cell.Updates.AddLast(info);
```

Comment:
Can this be simplified to `cell?.Updates.AddLast(info);`

Reply:
missed opportunity! yes